__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  connection.end();
+});
```

Comment:
you should be returning connection.end()

Reply:
added return